### DISCUSSION TITLE
Where do trashed files go in Linux?

### DISCUSSION THREAD

**Discussion by fritzrehde** [sxyazi/yazi#491] [1. Q&A] [answered]
Hi, I think `yazi` is using the `trash` crate to trash files. How does this work in Linux? Are trashed files put into a certain directory? Given that this feature exists in yazi, I would appreciate if there were a little more documentation on e.g. the difference between trashing and deleting files, maybe with links to how trashing works on the different supported OS's. 
Thanks!

**Comment by fritzrehde** [comment]
Ok, having done some research myself, this seems to be the `.local/share/Trash` directory on my system. I still think some small documentation additions would be beneficial.

**Comment by sxyazi** [comment, marked as answer]
Yes, it uses the FreeDesktop.org Trash specification, see https://docs.rs/trash/3.1.2/trash/#notes-on-the-linux-implementation. 

I added it alongside the remove command. https://yazi-rs.github.io/docs/configuration/keymap/#:~:text=if%20it%20exists.-,remove,-%3A%20Move%20the%20files
